frontend/src/components/VBackToSearchResultsLink.vue

```diff
@@ -3,14 +3,18 @@
   <VLink
     class="time inline-flex flex-row items-center gap-2 rounded-sm p-2 text-xs font-semibold text-dark-charcoal-70 pe-3 hover:text-dark-charcoal"
     v-bind="$attrs"
+    @click="handleClick()"
```

Comment:
When referring to an event handler function, the parenthesis are not used (because that would invoke the function).

```suggestion
    @click="handleClick"
```